Uanion/Uanion.Application/Features/User/Commands/DeleteUser/DeleteUserCommand.cs

```diff
@@ -0,0 +1,9 @@
+﻿using MediatR;
+
+namespace Uanion.Application.Features.User.Commands.DeleteUser;
```

Comment:
Need to add empty line between namespace and class.

Reply:
Done